_rules/heading-descriptive-b49b2e.md

```diff
@@ -27,21 +27,13 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [semantic][semantic role] `heading` element that is either [visible][] or [included in the accessibility tree][].
+This rule applies to any [semantic][semantic role] `heading` element that is [included in the accessibility tree][] and has a non-empty (`””`) [accessible name][].
 
-## Expectation 1
+## Expectation
 
-For each target element at least one of the following is true:
+Each target element describes the topic or purpose of the first [perceivable content][] after the test target that is not [decorative][].
```

Comment:
```suggestion
Each target element describes the topic or purpose of the first [perceivable content][] after the test target that is not
- Only [decorative][] content
- A [semantic][semantic role] `heading` element that is [included in the accessibility tree][] and has a non-empty (`””`) [accessible name][]
```

Reply:
@giacomo-petri @carlosapaduarte This is more or less the fix you were requesting. However, what would happen if there was a sentence after the third accordion, but before the next heading? Then the heading would go back to failing and I'm not sure that would seem correct either.

Reply:
Agreed @tbostic32 The problem remains. Shall we add an assumption that the content the heading is describing is not invisible and hidden from the AT? Otherwise, I believe we will end with with a fairly complex rule...